src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -911,6 +912,8 @@ public static void assertRejected(Whitelist whitelist, String expectedSignature,
             // GroovyRuntimeException.
             if (x.getCause() instanceof RejectedAccessException) {
                 errors.checkThat(x.getMessage(), ((RejectedAccessException)x.getCause()).getSignature(), is(expectedSignature));
+            } else {
+                errors.addError(x);
```

Comment:
This corrects a mistake in 85d16bd851b355458f6396dbc4f92c3e0eebcd72. Thankfully it did not mask any existing issues.